client/src/controllers/RevealController.ts

```diff
@@ -142,10 +144,12 @@ export class RevealController extends Controller<HTMLElement> {
 
   close() {
```

Comment:
```suggestion
  /**
   * Close (hide) the reveal content.
   */
  close() {
```

let's leave the code a bit better documented.